io/net/bonding.py

```diff
@@ -45,13 +45,16 @@ def setUp(self):
         depends = []
         if detected_distro.name == "Ubuntu":
             depends.append("openssh-client")
+            depends.append("iputils-ping")
         # FIXME: "redhat" as the distro name for RHEL is deprecated
         # on Avocado versions >= 50.0.  This is a temporary compatibility
         # enabler for older runners, but should be removed soon
         if detected_distro.name in ["rhel", "fedora", "centos", "redhat"]:
             depends.append("openssh-clients")
+            depends.append("iputils")
         if detected_distro.name == "SuSE":
             depends.append("openssh")
+            depends.append("iputils")
```

Comment:
can this be done, instead of two append every where.

>>> a = []
>>> a.extend(["harsha", "venkat", "sim", "praveen"])
>>> print a
['harsha', 'venkat', 'sim', 'praveen']
>>>